lib/elastic_record/index/documents.rb

```diff
@@ -27,20 +30,19 @@ def request_more_hits
       end
 
       def request_next_scroll
-        if scroll_id.nil?
-          response = initial_search_response
-        else
-          response = @elastic_index.scroll(scroll_id, keep_alive)
-        end
-
-        @scroll_id = response['_scroll_id']
+        response = scroll_ids.any? ? @elastic_index.scroll(scroll_ids.last, keep_alive) : initial_search_response
```

Comment:
scroll_id became scroll_ids which should always be exist because of the initialize line. It is kind of confusing because I kept the signature as is, but that's mostly because it would only ever make sense in the current usage to pass a sing scroll id. (At least how it's used now)

Reply:
I see.  In the line below this one, is it possible `response['_scroll_id']` will be `nil` or non-unique?  Might need to re-compact the scroll_ids if so.

Reply:
I don't think it would ever be nil (Barring a error or something, which will raise an error, so doesn't really matter), but not unique certainly according the ES docs. But that's ok, because the DELETE endpoint is very permissive. You can send it duplicates/expired scroll ids, etc and it just tells you how many it actually cleared. We just want to make sure the last result is always the end of that array for the next usage.